janitor/utils.py

```diff
@@ -869,8 +870,8 @@ def _create_indexer_for_complete(
 
 def _complete_indexer_expand_grid(indexer):
     """
-    Generate indices to expose explicitly missing values,
-    using the `expand_grid` function.
+    Generate indices to expose explicitly missing values, using the
+    `expand_grid` function.
```

Comment:
We should reinstate the original as the original docstring follows SEMBR. https://sembr.org